src/tests/ftest/pool/list_pools.py

```diff
@@ -35,47 +36,71 @@ def run_case(self, rank_lists, svcn=None):
-        # information.  The DmgCommand.pool_info() method returns the command
-        # output as a dictionary of pool UUID keys with service replica list
-        # values.
-        detected_uuids = {}
+            expected_admin_uuids[self.pool[-1].uuid.lower()] = self.pool[-1].svc_ranks
+
+        # Remove the default ACLs that allow the creator to access the pool.
+        # These ACLs don't apply to dmg, but do apply to users.
+        offlimits = self.pool[0]
+        self.get_dmg_command().pool_delete_acl(offlimits.uuid, 'OWNER@')
+        self.get_dmg_command().pool_delete_acl(offlimits.uuid, 'GROUP@')
+        expected_user_uuids = expected_admin_uuids.copy()
+        del expected_user_uuids[offlimits.uuid.lower()]
+
+        # Verify the 'dmg pool list' command lists the correct created pool
+        # information.
+        detected_admin_uuids = {}
         try:
             for data in self.get_dmg_command().get_pool_list_all():
-                detected_uuids[data["uuid"]] = data["svc_reps"]
+                detected_admin_uuids[data["uuid"]] = data["svc_reps"]
         except KeyError as error:
             self.fail("Error parsing dmg pool list output: {}".format(error))
 
-        self.log.info("Expected pool info: %s", str(expected_uuids))
-        self.log.info("Detected pool info: %s", str(detected_uuids))
-
-        # Destroy all the pools
-        if self.destroy_pools(self.pool):
-            self.fail("Error destroying pools")
-        self.pool = []
+        self.log.info("Expected admin pool info: %s", str(expected_admin_uuids))
+        self.log.info("Detected admin pool info: %s", str(detected_admin_uuids))
 
         # Compare the expected and detected pool information
         self.assertEqual(
-            expected_uuids, detected_uuids,
+            expected_admin_uuids, detected_admin_uuids,
             "dmg pool info does not list all expected pool UUIDs and their "
             "service replicas")
 
+        # Verify the 'daos pool list' command lists the correct created pool
+        # information.
+        detected_user_uuids = {}
+        try:
+            for data in self.get_daos_command().get_pool_list_all():
+                detected_user_uuids[data["uuid"]] = data["svc_reps"]
+        except KeyError as error:
+            self.fail("Error parsing dmg pool list output: {}".format(error))
+
+        self.log.info("Expected user pool info: %s", str(expected_user_uuids))
+        self.log.info("Detected user pool info: %s", str(detected_user_uuids))
+
+        # Compare the expected and detected pool information
+        self.assertEqual(
+            expected_user_uuids, detected_user_uuids,
+            "daos pool info does not list all expected pool UUIDs and their "
+            "service replicas")
+
+        # Destroy all the pools
+        if self.destroy_pools(self.pool):
+            self.fail("Error destroying pools")
+        self.pool = []
```

Comment:
This is pre-existing, but I'm not sure why we even need this at the end of the test since teardown will destroy all the pools. But I'm not 100% confident there isn't some implicit assumption somewhere

Reply:
Unless you feel strongly about it, I'm inclined to leave well enough alone, at least in this PR.